packages/expo-age-range/src/AgeRange.ts

```diff
@@ -1,13 +1,3 @@
 import ExpoAgeRange from './ExpoAgeRange';
-import { AgeRangeRequest, AgeRangeResponse } from './ExpoAgeRange.types';
 
-/**
- * Prompts user to declare their age range.
- * @return A promise that resolves with user's age range response. Or rejects with an error (TODO vonovak export the error codes?).
- * @platform ios 26.0+
- * @platform android
- */
-export async function requestAgeRangeAsync(options: AgeRangeRequest): Promise<AgeRangeResponse> {
-  // TODO vonovak can just export the function directly?
-  return ExpoAgeRange.requestAgeRangeAsync(options);
-}
+export const requestAgeRangeAsync = ExpoAgeRange.requestAgeRangeAsync;
```

Comment:
We shouldn't do this in the top-level scope. `ExpoAgeRange` is a host object that lazily initializes the native module under the hood on the first property access. If you do it in the top-level scope, it's basically loaded eagerly. I think we should just export `ExpoAgeRange` as default